### ISSUE TITLE
Generate default config by OS

### ISSUE THREAD

**Issue by Hikari9** [Hikari9/comfortable-swipe#87]
## Summary

The plan is to dispatch default config by OS / Desktop Environment. Will revolve around workspace switching most likely.

## Related Issues

As suggested by @neerajvashistha:
> @Hikari9 thank you for such an amazing repo and such a clean way to implement this. But a suggestion, if you can recognise the Linux distribution and then generate the config.

_Originally posted by @neerajvashistha in https://github.com/Hikari9/comfortable-swipe/issues/76#issuecomment-619420439_

## Desktop Environment Checklist
* [X] Ubuntu Unity
  
  Condition:
  ```
  [[ "$DESKTOP_SESSION" == unity ]]
  ```

  Config:
  ```
  left3  = ctrl+alt+Right
  left4  = ctrl+alt+shift+Right
  right3 = ctrl+alt+Left
  right4 = ctrl+alt+shift+Left
  up3    = ctrl+alt+Down
  up4    = ctrl+alt+shift+Down
  down3  = ctrl+alt+Up
  down4  = ctrl+alt+shift+Up
  threshold = 0.0
  ```

* [ ] GNOME
* [ ] KDE
* [ ] Elementary OS
* [ ] Kali Linux
* [ ] Pop OS GNOME (new)
  ```
  up3 = Ctrl+Super+Down  # workspace down
  down3 = Ctrl+Super+Up   # workspace up
  left3 = Ctrl+Super+Right   # workspace right
  right3 = Ctrl+Super+Left   # workspace left
  up4 = Super+S  # show workspaces
  down4 = Super+D  # show desktop
  left4 = ???
  right4 = ???
  threshold = 1.0
  ```

**Comment by Hikari9** [comment]
We can probably mimic defaults from `fusuma` in https://github.com/iberianpig/fusuma/wiki/PopOS-Default-Gnome

**Comment by AlwaysBeCalm** [comment]
@Hikari9 
I have a suggestion
it would be very excellent if you make the config file get it's button from the default shortcuts of the system

e.g
In POP!_OS to move to below workspace, the user click <pre>Ctrl+Super+Down</pre> and in Ubuntu it's <pre>Super+Page Down</pre> or <pre>ctrl+alt+down</pre>

if you make the config file recognize the OS then puts the default buttons (by getting it from the system default) on every swipe.

I hope it's clear :sweat_smile:

**Comment by Hikari9** [comment]
Yes that's the intention of this issue 👍 So far there's config for ubuntu and PopOS above in the checklist. Thanks for your great contributions to our issue threads @AlwaysBeCalm you are awesome :rocket:

**Comment by AlwaysBeCalm** [comment]
you welcome :rose: 
@Hikari9 Keep up the best work :smile:
